sections/About2.tsx

```diff
@@ -0,0 +1,82 @@
+import { Col, Container, Row, Image } from "react-bootstrap";
+import { ButtonProps, Heading, Text, Link, Button } from "../components";
+import styled from "styled-components";
+import media from "../theme/media";
+
+type About1Props = {
```

Comment:
Probably rename `About1` to `About2` throughout the file, even though it makes no difference